docs/blog/2020-04-13-upgrading-to-jamstack-with-agility/index.md

```diff
@@ -29,12 +29,12 @@ For those of you who just want to see the code for the new site (it only has cod
 
 Here's what I did to get everything up and running, right from coding the new site, to deploying, testing and flipping over the DNS.
 
-Get it running locally with Gatsby
-Implement the Header and Footer
-Create a new Home Page
-Run it in Gatsby Cloud
-Deploy to Netlify
-Setup the CDN to do the Edge Routing
+[Get it running locally with Gatsby](#step-1-get-it-running-locally-with-gatsby)
+[Implement the Header and Footer](#step-2-implement-the-header-and-footer)
+[Create a new Home Page](#step-3-create-a-new-home-page)
+[Run it in Gatsby Cloud](#step-4-run-it-in-gatsby-cloud)
+[Deploy to Netlify](#step-5-deploy-to-netlify)
+[Setup the CDN to do the Edge Routing](#step-6-setup-the-cdn-to-do-the-edge-routing)
```

Comment:
I think this should be a list?

```suggestion
- [Get it running locally with Gatsby](#step-1-get-it-running-locally-with-gatsby)
- [Implement the Header and Footer](#step-2-implement-the-header-and-footer)
- [Create a new Home Page](#step-3-create-a-new-home-page)
- [Run it in Gatsby Cloud](#step-4-run-it-in-gatsby-cloud)
- [Deploy to Netlify](#step-5-deploy-to-netlify)
- [Setup the CDN to do the Edge Routing](#step-6-setup-the-cdn-to-do-the-edge-routing)
```

Reply:
ah yes - that was the initial idea :)
fixed